certified-connectors/UiPath Orchestrator/apiProperties.json

```diff
@@ -0,0 +1,85 @@
+{
+  "properties":
+  {
+    "connectionParameters":
+    {
+      "token":
+      {
+        "type": "oauthSetting",
+        "oAuthSettings":
+        {
+          "identityProvider": "oauth2",
+          "clientId": "[enter client id]",
+          "scopes":
+          [
+            "OR.Execution.Read OR.Folders.Read OR.Jobs.Write OR.Queues offline_access"
```

Comment:
I expect this to be a JSON array of strings like the following. Can you please change to a JSON array and retest?
https://github.com/microsoft/PowerPlatformConnectors/blob/5a304e21c47b594bc18500252ae9f63634956d85/certified-connectors/Adobe%20Creative%20Cloud%20Libraries/apiProperties.json#L9

Reply:
@mamurshe Works with string array, thanks.